titus-server-master/src/main/java/com/netflix/titus/master/taskmigration/V3TaskMigrationDetails.java

```diff
@@ -156,7 +157,7 @@ public void setMigrationDeadline(long migrationDeadline) {
                         return Optional.of(serviceTask.toBuilder().withMigrationDetails(newMigrationDetails).build());
                     }
                     return Optional.empty();
-                }, Trigger.TaskMigration, "Updating migration details", CallMetadata.newBuilder().withCallerId("task migrations").withCallReason("migration").build()).await(30_000, TimeUnit.MILLISECONDS);
+                }, Trigger.TaskMigration, "Updating migration details", JobManagerConstants.SCHEDULER_CALLMETADATA.toBuilder().withCallReason("task migration").build()).await(30_000, TimeUnit.MILLISECONDS);
```

Comment:
Caller here is task migration component

Reply:
The latest commit should have this change